c_src/sqlite3_nif.c

```diff
@@ -12,6 +12,36 @@
 
 #define MAX_ATOM_LENGTH 255
 
+static ERL_NIF_TERM am_ok;
+static ERL_NIF_TERM am_error;
+static ERL_NIF_TERM am_nil;
+static ERL_NIF_TERM am_out_of_memory;
+static ERL_NIF_TERM am_done;
+static ERL_NIF_TERM am_row;
+static ERL_NIF_TERM am_rows;
+static ERL_NIF_TERM am_invalid_filename;
+static ERL_NIF_TERM am_invalid_flags;
+static ERL_NIF_TERM am_database_open_failed;
+static ERL_NIF_TERM am_failed_to_create_mutex;
+static ERL_NIF_TERM am_invalid_connection;
+static ERL_NIF_TERM am_sql_not_iolist;
+static ERL_NIF_TERM am_connection_closed;
+static ERL_NIF_TERM am_invalid_statement;
+static ERL_NIF_TERM am_invalid_chunk_size;
+static ERL_NIF_TERM am_busy;
+static ERL_NIF_TERM am_invalid_column_count;
+static ERL_NIF_TERM am_transaction;
+static ERL_NIF_TERM am_idle;
+static ERL_NIF_TERM am_database_name_not_iolist;
+static ERL_NIF_TERM am_serialization_failed;
+static ERL_NIF_TERM am_deserialization_failed;
```

Comment:
I haven't used this API before, but I think it might also benefit from being rc + exception.